source/reference.txt

```diff
@@ -2,8 +2,11 @@
 Reference
 =========
 
+:ref:`about-mongosync`
+   The primary process used in ``mongosync``.
+
 :ref:`c2c-mongosync`
```

Comment:
[suggestion] Since it sounds like the "About mongosync" page will be where we primarily want to direct users for learning about mongosync, I think it'd be worth nesting the current mongosync page under that. LMK if you aren't sure how to do that!

Reply:
Amazing, that sounds good.